lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -202,7 +204,7 @@ def _prepare_source_vhd(self, node_runbook: HypervNodeSchema) -> None:
         elif self._source_files:
             for artifact_path in self._source_files:
                 if artifact_path.suffix == ".vhd" or artifact_path.suffix == ".vhdx":
-                    self._source_vhd = artifact_path
+                    self._source_vhd = PureWindowsPath(artifact_path)
```

Comment:
All used place can be PurePath. `PureWindowsPath` is used only when initializing.

Reply:
Done. Using PurePath everywhere.